### PR TITLE
doc: add doc about basic perf testing at initial demo environment

### PR DESCRIPTION
**Pull request by ideepika**
install

### CODE REVIEW

File: perf_tests/perf_testing_basic.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Doc: demo system
+
+Created by: Deepika Upadhyay
+Created time: November 7, 2023 5:33 PM
```
Comment (zalsader) [inline perf_tests/perf_testing_basic.md]:
```suggestion
Created by: Deepika Upadhyay

Created time: November 7, 2023 5:33 PM
```

**Review by zalsader — APPROVED** [review]
Looks good!

File: perf_tests/perf_testing_basic.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+commonly used commands have been adapted to this script.
+
+[https://github.com/koor-tech/demo-gitops/blob/main/perf_tests/test-fio.sh](https://github.com/koor-tech/demo-gitops/blob/main/perf_tests/test-fio.sh) 
+
+## Network Performance using Ancientt:
+
+This step should be performed after you have installed the Kubernetes cluster, we use [ancientt](https://github.com/galexrt/ancientt) tool, which uses iperf3 to perform cross node network testing.
+
+- please refer the installation guide to install ancientt on you development enviroment,
```
Comment (galexrt) [inline perf_tests/perf_testing_basic.md]: Let's link to the install guide of `ancientt` so it is easy to run the tests

Comment (ideepika) [reply perf_tests/perf_testing_basic.md]: good point, updated, thanks!

**Pull request merged**